src/librustdoc/passes/non_autolinks.rs

```diff
@@ -12,7 +12,7 @@ use rustc_session::lint;
 crate const CHECK_NON_AUTOLINKS: Pass = Pass {
     name: "check-non-autolinks",
     run: check_non_autolinks,
-    description: "detects URLS that could be written using angle brackets",
+    description: "detects URLs that could or should be written using angle brackets",
```

Comment:
Not sure if the `or should` part should be added here...

Reply:
I think I feel confused about the lint. What is the purpose of it? I hadn't heard about it before working on https://github.com/rust-lang/rustc-dev-guide/pull/1048.

Reply:
Markdown provides a syntax to transform a URL into a link:

```
<url>
```

If we see a URL without it, we lint saying that it could be turned into a link.

Reply:
Ah, I didn't realize that rustdoc didn't linkify bare links (without the angle brackets).

Reply:
Well, we handle markdown. ;)